im.kaidan.kaidan.json

```diff
@@ -50,13 +50,13 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://github.com/qxmpp-project/qxmpp/archive/refs/tags/v1.10.2.tar.gz",
-                    "sha256": "5d88a0841fe4543c84b843b329fa2a9560fd709f4cd0a1c4bf8089f6ab66c58d",
+                    "url": "https://invent.kde.org/libraries/qxmpp/-/archive/v1.10.2/qxmpp-v1.10.2.tar.gz",
```

Comment:
No, you can not link to invent.kde.org

Please put the tarball in the proper downoads.kde.org place

Reply:
That is not used by QXmpp (yet).

Reply:
We don't want Ben to ban qxmpp now that you have just joined us, so we will not be merging this.

Get the downloads.kde.org place fixed and then update this MR :)